### PR TITLE
Video streaming tutorial

### PR DESCRIPTION
**Pull request by AlexeyBulatov**
How to stream using MJPG-streamer.

### CODE REVIEW

File: docs/examples/camera.md [hunk 1]
```diff
@@ -41,3 +41,36 @@ Take photo:
...
+
+## Video streaming
+
+For a start you need to configure network: [Wi-Fi](../connectivity/wifi), [Ethernet](../connectivity/ethernet) or [USB-Ethernet](../connectivity/usb-ethernet).
+
+For video streaming you can use mjpg-streamer, which recipe you can find in meta-emlid-neutis-examples.
+
+MJPG-streamer takes JPGs from Linux-UVC compatible webcams, filesystem or other input plugins and streams them as M-JPEG via HTTP to webbrowsers, VLC and other software. It is the successor of uvc-streamer, a Linux-UVC streaming application with Pan/Tilt.
+
+Start stream with the following command:
```
Comment (staroselskii) [inline docs/examples/camera.md]: Start streaming

File: docs/examples/camera.md [hunk 2]
```diff
@@ -41,3 +41,36 @@ Take photo:
 ```bash
 fswebcam -d /dev/video0 -r 640x480 image.jpg
 ```
+
+## Video streaming
+
+For a start you need to configure network: [Wi-Fi](../connectivity/wifi), [Ethernet](../connectivity/ethernet) or [USB-Ethernet](../connectivity/usb-ethernet).
```
Comment (staroselskii) [inline docs/examples/camera.md]: First of all

Comment (staroselskii) [reply docs/examples/camera.md]: using any method described in [../connectivity/connectivity]

File: docs/examples/camera.md [hunk 3]
```diff
@@ -41,3 +41,36 @@ Take photo:
...
+
+For video streaming you can use mjpg-streamer, which recipe you can find in meta-emlid-neutis-examples.
```
Comment (staroselskii) [inline docs/examples/camera.md]: For video streaming, you can use mjpg-streamer. You can find the recipe in [meta-emlid-neutis-examples](link)

File: docs/examples/camera.md (hunk 2 above)
Comment (AlexeyBulatov) [reply docs/examples/camera.md]: We can't stream using Serial

**Pull request merged**